distsql/select_result.go

```diff
@@ -231,16 +234,31 @@ func (r *selectResult) readRowsData(chk *chunk.Chunk) (err error) {
 		}
 	}
 	r.selectResp.Chunks[r.respChkIdx].RowsData = rowsData
+	r.memConsume(-int64(originMemory - r.selectResp.Size()))
```

Comment:
r.memConsume(-int64(r.selectResp.Size() - originMemory))

Reply:
I prefer to directly to use `-`, it indicates a memory releasing.

Reply:
You're right. I'll try to implement in another way.